acceptance-amazon-rds-sync/import-table-fields.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Local directory
+DIR=$(dirname "$0")
+
+# Include common RDS tasks
+source $DIR/config
+
+echo "Restoring backed up fields values"
+for i in *.field_backup.sql
+do
+  echo "Importing $i"
+  mysql -h $DATABASE_HOST $DATABASE_NAME < $i
```

Comment:
might be worth checking mysql exit code (`if [ $! -gt 0 ]; then.... `) here, because when mysql fails, you will loose the field backup file forever.